build/dev/Dockerfile

```diff
@@ -43,8 +44,8 @@ WORKDIR /home/developer/kubefirst
 COPY --chown=developer:developer . .
 
 # download dependencies and prepare fresh installation
-RUN go mod download &&  \
-    go build
+RUN go mod download
+RUN go build -ldflags="-X github.com/kubefirst/kubefirst/configs.K1Version=${KUBEFIRST_VERSION}" .
 
```

Comment:
This separation, make local test much faster by allowing cache of remote libs. And preventing docker to re-download it.